### PR TITLE
pass tweet attributes through on blockquotes

### PR DESCRIPTION
**Pull request by andygnewman**
cc: @georgecrawford @i-like-robots 

For blockquotes;
- passes through existing classes (eg. twitter-tweet)
- adds id of tweet as data-tweet-id attribute

### CODE REVIEW

**Comment by georgecrawford** [comment]
Thanks Andy. LGTM.

File: tests/server/stylesheets/blockquotes-spec.js [hunk 1]
```diff
@@ -16,11 +16,28 @@ describe('Blockquotes', function () {
 			, 'main')
 			.then(function (transformedXml) {
 				transformedXml.should.equal(
-					'<blockquote class="n-content-blockquote">' +
+					'<blockquote class="n-content-blockquote ">' +
```
Comment (i-like-robots) [inline tests/server/stylesheets/blockquotes-spec.js]: a good example for switching to HTML assertions rather than string matching ;-]

**Comment by roland-vachter** [comment]
+1

**Comment by i-like-robots** [comment]
So post discussion 6 months later I guess we're going for appending the class name `n-content-blockquote--tweet` if the original `<blockquote>` has the class name `twitter-tweet`. That sound right?

**Comment by andygnewman** [comment]
Yep, have retained the original twitter-tweet class aswell in case we every use the twitter widget to embed the official rendering as per https://dev.twitter.com/web/embedded-tweets.

**Pull request merged**